.github/workflows/pull_request.yml

```diff
@@ -1,18 +1,39 @@
+name: Build Chain
+
 on: [ workflow_dispatch ]
 
 jobs:
-  build-chain:
+  build-chain-openjdk8:
+    runs-on: ubuntu-latest
+    name: Pull Request openjdk8
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Chain
+      id: build-chain
+      uses: kiegroup/github-action-build-chain@openjdk8
+      with:
+        parent-dependencies: 'drools@7.x:master'
+        child-dependencies: 'jbpm@7.x:master'
```

Comment:
Is there any doc I could read on how this particular config works? I'm confused by both "7.x" and "master" being there.

Reply:
look into the GA extension https://github.com/kiegroup/github-action-build-chain I think @Ginxo will add documentation as it goes. Right now fast speed implementation :)

Reply:
If I understand it correctly, according to [1], it says our `7.x` depends on the other repo's `master` branch.

[1] https://github.com/kiegroup/github-action-build-chain/blob/master/src/lib/common.js#L65

Reply:
yes, the documentation still missing https://issues.redhat.com/browse/BXMSPROD-913 sorry for that
but basically whatever comes after `@` means `branch mapping`, so it will map the current `7.x` branch to jbpm/drools `master`
The same here but opposite way https://github.com/kiegroup/jbpm/blob/5e3cb75e2111668d4e093605f5c82df2b9c53183/.github/workflows/pull_request.yml#L15

Reply:
> If I understand it correctly, according to [1], it says our `7.x` depends on the other repo's `master` branch.
> 
> [1] https://github.com/kiegroup/github-action-build-chain/blob/master/src/lib/common.js#L65

this is what Optaplanner with having 8.x version on its master made up ;) @rsynek